### PR TITLE
React with styled component

### PR DESCRIPTION
**Pull request by pankajsoni91**
Fixes for :
Added the styled component 
Absolute Path from root dir 

- will be working on other feedback suggested by you.

### CODE REVIEW

File: src/client/index.tsx [hunk 1]
```diff
@@ -1,8 +1,14 @@
 import * as React from 'react';
 import { render } from 'react-dom';
-import App from './routes/Routes';
+import Routes from './routes/Routes';
 import { BrowserRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
```
Comment (kuldeepkeshwar) [inline src/client/index.tsx]: Provider is not required  here

please add the rule for unused object

File: src/client/routes/dashboard/index.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import { Wrapper } from 'client/components/Wrapper';
+import { Button } from 'client/components/Button';
+
+interface IDashboardState {
+  counter: number;
+}
+
+interface IDashboardProps {
+  initCounter: number;
```
Comment (kuldeepkeshwar) [inline src/client/routes/dashboard/index.tsx]: name should be counter

File: src/client/routes/dashboard/index.tsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import { Wrapper } from 'client/components/Wrapper';
+import { Button } from 'client/components/Button';
+
+interface IDashboardState {
```
Comment (kuldeepkeshwar) [inline src/client/routes/dashboard/index.tsx]: name should be IState, IProps

File: src/client/index.tsx (hunk 1 above)
Comment (pankajsoni91) [reply src/client/index.tsx]: Working on the ts-lint issue. can you suggest where I will get the all the required linting configuration?

File: src/client/routes/dashboard/index.tsx (hunk 2 above)
Comment (pankajsoni91) [reply src/client/routes/dashboard/index.tsx]: Done

File: src/client/routes/dashboard/index.tsx (hunk 3 above)
Comment (pankajsoni91) [reply src/client/routes/dashboard/index.tsx]: Done

**Comment by kuldeepkeshwar** [comment]
LGTM 😃

**Comment by pankajsoni91** [comment]
Thank You.

**Pull request merged**